jetson/percep/pcl.cpp

```diff
@@ -236,26 +237,44 @@ double PCL::FindClearPath(const std::vector<std::vector<int>> &interest_points,
     
     //Check Center Path
     if(CheckPath(interest_points, viewer, obstacles, compareLine(0,-HALF_ROVER), compareLine(0,HALF_ROVER))) {
-        std::cout << "CENTER PATH IS CLEAR!!!" << std::endl;
+            std::cerr << "CENTER PATH IS CLEAR!!!" << std::endl;
+            std::cerr << "Distance from Last Obstacle: " << distance << " meters" << std::endl; 
         return 0;
     }
+    else { //otherwise, check to find left and right paths
+
+        //Values that store the distances of the last obstacle from a given CheckPath
+        double centerDistance, leftDistance, rightDistance;
+        centerDistance = distance; //from previous loop of CheckPath, where it was only looking at the center path for a clear path
+
+        //Initialize base cases outside of scope
+        vector<int> centerObstacles = {obstacles.at(0), obstacles.at(1)};
 
-    //Initialize base cases outside of scope
-    vector<int> centerObstacles = {obstacles.at(0), obstacles.at(1)};
+        //Find Clear left path
+        std::cerr << "Checking Left Path" << std::endl;
```

Comment:
Can you change cerrs to couts?

Reply:
Just a consistency thing we're trying to do. Only change them in the ones in this function